modules/react/modal/stories/examples/BodyOverflow.tsx

```diff
@@ -43,7 +42,7 @@ export const BodyOverflow = () => {
             <p>Are you sure you want to delete the item?</p>
             <p>Are you sure you want to delete the item?</p>
           </Modal.Body>
-          <HStack spacing="s" paddingTop="s">
+          <HStack spacing="s" paddingTop="s" paddingX="l" paddingBottom="l">
```

Comment:
Now that we've removed `Modal.Footer`, it feels odd to me to require consumers to know they need `paddingX="l"` and `paddingBottom="l"` to get their `HStack` Footer to render with complementary padding to match `Modal.Body` and `Modal.Heading`. Previously, this was padding was baked into `Modal.Footer`.

Reply:
if we make this a Minor release, it makes sense to add this back in. But I agree the consumer would have to know what padding to add

Reply:
For now, the consumer will need to add their own footer (using an `HStack`, for instance) with the appropriate paddings and margins to align with the Heading and Body subcomponents (see screenshot in top-level PR comment). We'll update Popup, Modal, and Dialog examples to include example footers with the needed padding and margin.